src/content/docs/en/user-guide/bridge.mdx

```diff
@@ -20,32 +20,25 @@ excerpt: "To start bridging assets from Sepolia, navigate to the portal bridge a
 [^thanks-hop]: Forked from [Hop Exchange](https://hop.exchange/)'s UI 🙌
 
-Deposits can take up to 15 minutes to be available on Scroll.
-
-Withdrawals, which require a second interaction on Sepolia after the withdrawal is finalized, can take much longer.
-
-:::caution[Waiting a long time for a bridge transaction?]
-The time estimates above are typical for normal network behavior and activity levels. Because we only process so many queued L1 transactions per an L2 block, bridged messages may take longer to be included on Scroll in times of exceptional network usage.
-:::
-
-## Deposit from Sepolia to Scroll Sepolia
+## Deposit from L1 to L2
 
 ### Instructions
 
-1. First, navigate to the [Scroll Bridge](https://sepolia.scroll.io/bridge) and press the "Connect Wallet" button.
-1. In the app, ensure that **Ethereum Sepolia** is on top and **Scroll Sepolia** is on the bottom. You can click the "**↓**" button to switch their positions.
-1. Select the token you want to transfer from Sepolia to Scroll Sepolia. If it's you're first time bridging, try "ETH."
-1. If this is your first time transferring a specific ERC20 token, you must **Approve** the Sepolia Bridge contract to access your ERC20 token.
-1. Next, click the **Send to Scroll Sepolia** button to make the deposit. Your wallet will ask to confirm the transfer transaction.
-1. Once the transfer transaction is sent and confirmed, the token will be deducted from your Sepolia wallet.
-1. You can always check the status of a transaction by pressing the "History" icon next to your wallet address in the top-right corner.
+1. After navigating to the bridge app, press the "Connect Wallet" button. You might need to switch your wallet to the right network.
+1. In the app, select the **Deposit to Scroll** tab.
+1. Select the token you want to transfer from the L1 network. If it's your first time bridging, try "ETH."
+1. If this is your first time transferring a specific ERC20 token, you must **Approve** the Sepolia or Ethereum Bridge contract to access your ERC20 token.
+1. Select your deposit mode. The Fast option initiates a bridge transfer immediately, while the Economy option groups multiple requests together and initiates a single bridge transfer for the batch, sharing the cost among all included requests.
+1. Next, slide the **Deposit funds** button to make the deposit. Your wallet will ask to confirm the transfer transaction.
+1. Once the transfer transaction is sent and confirmed, the token will be deducted from your wallet.
+1. You can always check the status of a transaction by pressing the "Transaction History" icon next to your wallet address in the top-right corner.
 
-### When will the token arrive in your Scroll Sepolia wallet?
+{/* ### When will the token arrive in your Scroll Sepolia wallet?
```

Comment:
We still need something to answer the FAQ of how long bridging takes. It's clear below for L2->L1, but not here, where we actually need to give a rough estimate of "Fast" v "Economy"

Reply:
In the UI, when you select between Fast or Economy it tells the user the estimated amount of time each option will take